pkg/generator/templates/cloud-init-ubuntu.template

```diff
@@ -36,7 +36,7 @@ write_files:
 - path: '{{ $dropIn.Path }}'
   encoding: b64
   content: |
-    {{ $dropIn.Content }}
+    {{printf "%s\n" $dropIn.Content}}
 {{- end -}}
 {{- end -}}
```

Comment:
```suggestion
{{- end }}
```

Reply:
I removed the dashes as suggested on line 40 and 41. Only added a new line after the unit and did not resolve the bug for the drop/in units. I am not too familiar with the go template format, but that was the suggested approach to start on a new line.